.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+
+  - package-ecosystem: docker
+    directory: /cluster/images/provider-opentofu
+    schedule:
+      interval: daily
+
+  - package-ecosystem: gomod
```

Comment:
it looks like this relies on Dependabot for security updates. Unfortunately, this doesn't help us with release branches when updating gomod: https://github.com/dependabot/dependabot-core/issues/2767.

for automated security updates across multiple branches, we could adopt what c/c is doing: https://github.com/crossplane/crossplane/blob/05a7fe1a41993f0ea7f0f24dc1f6dacffb73cdab/.github/renovate.json5#L15

Reply:
This effort isn't about managing CVEs but more about hardening our workflows to prevent/mitigate the impact of the recent security incident with tjactions.

Reply:
Yep -- I was referring to this specific file's configuration with that comment even if security updates are out of scope. The rest of the changeset looks reasonable to me.

Reply:
Ah gotcha 👍